website/integrations/services/beszel/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Integrate with Beszel
+sidebar_label: Beszel
+---
+
+# Integrate With Beszel
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Beszel
+
+> Beszel is a lightweight server monitoring platform that provides Docker statistics, historical data, and configurable alerts.
+> It features a user-friendly web interface, supports multi-user management, OAuth authentication, and offers a REST API for integration with other applications.
+> Designed for simplicity, Beszel is easy to set up and operates efficiently without requiring public internet exposure.
+>
+> -- https://beszel.dev/
+>
+> This guide explains how to configure Beszel to use authentik as the OAuth provider for logging in to the Web GUI.
+
+## Preparation
+
+Since Beszel uses PocketBase as its backend, you only need to replace the PocketBase placeholders with the placeholders found on this page.
```

Comment:
I think we need to more clearly explain the dependencies between PB and Beszel. And, even though it is not great to have to duplicate content, I don't think we should ask readers to have to go read other docs (the PB docs). I'll make a pass at a compromise idea.. we can see how it looks.

Question: does a user have to also configure PB as part of this Beszel setup? Or if they already set up PB before, do they not need to do anything?

Reply:
You install Beszel and it already comes with Pocketbase installed as its backend, no need for separate setup steps to get Pocketbase Running.

Reply:
oh oh, so two possibilities:

- IF they already have integrated PB, they still have to also do these steps for Beszel (basically the same steps as for PB but diff FQDNs of course), 

- if they have not yet installed PB, and they want to install Beszel, then they DO NOT have to also install PB also. It will come with the PB integration.

Is this correct @nicedevil007 ? Thank you for your bravery and patience... :-)

Reply:
this is correct, I followed the comment here to split those up,. If I have to revert back to the only Beszel Docs without pocketbase I have to say sorry, we close this then and I don't know who I can trust on advices here anymore :(

https://github.com/goauthentik/authentik/pull/12905#pullrequestreview-2587822774